components/studio/test/studio-from-source/test.ps1

```diff
@@ -0,0 +1,29 @@
+$env:HAB_LICENSE = "accept-no-persist"
+
+$env:studio_command = "bin/hab-studio.ps1"
+
+hab pkg install core/powershell
+hab pkg install core/7zip
+hab pkg install core/hab 
+hab pkg install core/hab-plan-build-ps1 
+
+mkdir "bin/powershell" | Out-Null
+mkdir "bin/hab" | Out-Null
+mkdir "bin/7zip" | Out-Null
+
+Copy-Item "$(hab pkg path core/powershell)/bin/*" "bin/powershell" -Recurse
+Copy-Item "$(hab pkg path core/hab)/bin/*" "bin/hab"
+Copy-Item "$(hab pkg path core/7zip)/bin/*" "bin/7zip"
+Copy-Item "$(hab pkg path core/hab-plan-build-ps1)/bin/*" "bin/"
+
+try {
+    test/shared/studio-enter.ps1 -studio_command "bin/hab-studio.ps1"
+    sleep 5
```

Comment:
do you need this sleep in addition to what is in `studio-enter.ps1`?

Reply:
I'll double check this to be certain it's still required, but what I was encountering was failures removing files in the `finally` block because the studio was still partially mounted. The sleep gave it a chance for any open handles to the copied files to be closed.  

If it's still required I'll add a comment to that effect, or if there's a better way to handle it I can implement that. Not a fan of arbitrary sleeps, but I'm not aware of another way on Windows to handle that.